doc/whats-new.md

```diff
@@ -139,42 +145,45 @@ quite significant, for instance the generation of ruptures for a large
 multipoint source for Colombia became 30x faster using 12x less memory.
 
 We changed the way ruptures are stored internally: the
-`code` field in the `ruptures` dataset is now an unique checksum
+`code` field in the `ruptures` dataset is now a unique checksum
 depending on the kind of rupture. Before it was an incremental
 number depending on the order of the Python module imports which
 was making debugging difficult.
 
-The rupture CSV exporter has been enhanced and now it exports the rupture
+The rupture CSV exporter has been enhanced, and now it exports the rupture
 surface boundaries as 3D multipolygons instead of 2D multipolygons.
 
-We fixed a small bug in the rupture XML exporter that was failing
+We fixed a small bug in the rupture XML exporter, which was failing
 if the user did not specify the hazard sites.
 
 We added the ability to generate hazard curves without storing the GMFs,
 simply by setting the flags
 ```
-  hazard_curves_from_gmfs=true
-  ground_motion_fields=true
```

Comment:
Is this correct? Shouldn't it be 
`hazard_curves_from_gmfs = true`
`ground_motion_fields = false` ?

Reply:
Yep, my mistake